src/boostvqe/boost.py

```diff
@@ -128,7 +127,6 @@ def dbqa_vqe(
         set_backend(backend=backend, platform=platform)
     else:
         set_backend(backend=backend)
```

Comment:
Above (in your previous comment) it was qiboml because in that specific function we are calculating gradients. To do so, we are using the [Tensorflow GradientTape](https://www.tensorflow.org/guide/autodiff) mechanism and for doing that we need a tensorflow backend.